### DISCUSSION TITLE
Cannot create a new domain

### DISCUSSION THREAD

**Discussion by hurricanetechnology** [simple-login/app#2222] [Q&A]
I tried to create a new domain in SL, but SL told me that that domain was already used in one of my SL mailboxes. While that was initially true, I had deleted that mailbox in SL that referenced that domain and then also set the default mailbox to another domain.
Still, however, SL won't let me create that domain and still insists that that domain is used in another of my SL mailboxes. It is not.
How to fix this?

Thanks,

**Comment by nguyenkims** [comment]
Note that the mailbox deletion can take some time as it requires to move all aliases owned by this mailbox to another mailbox. Does this problem persist if you retry now?
